Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -124,9 +124,31 @@ class BrowserViewController: UIViewController {
     }
 
     override func traitCollectionDidChange(previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
         updateToolbarState()
         showToolbars(animated: false)
```

Comment:
Do you think these should be moved too? I don't know much about this API -- what triggers a "trait collection" change?